vmdb/lib/vmdb/logging.rb

```diff
@@ -106,6 +106,9 @@ def self.create_loggers
     def self.configure_external_loggers
       require 'awesome_spawn'
       AwesomeSpawn.logger = $log
+
+      require 'manageiq_foreman'
+      ManageiqForeman.logger = $log
```

Comment:
This concerns me because logger configuration happens early.  Does the code have the manageiq_foreman piece required at this time?  If not, doesn't that sorta hurt lazy loading?  Perhaps this should live in the ProviderForeman somewhere instead?